superset-frontend/packages/superset-ui-core/test/chart/components/SuperChartCore.test.tsx

```diff
@@ -35,25 +30,11 @@ import {
   SlowChartPlugin,
 } from './MockChartPlugins';
 
-const Wrapper = ({
-  theme,
-  children,
-}: {
-  theme: SupersetTheme;
-  children: ReactNode;
-}) => <ThemeProvider theme={theme}>{children}</ThemeProvider>;
-
-const styledMount = (component: ReactElement) =>
-  mount(component, {
-    wrappingComponent: Wrapper,
-    wrappingComponentProps: {
-      theme: supersetTheme,
-    },
-  });
+const renderWithTheme = (component: ReactElement) =>
+  render(<ThemeProvider theme={supersetTheme}>{component}</ThemeProvider>);
```

Comment:
When importing `render` from `spec/helpers/testing-library`, the rendered component is already wrapped in theme provider 🙂

Reply:
You can never have too many theme providers, right? Matryoshka that thing all day long

```
<ThemeProvider theme={supersetTheme}>
  <ThemeProvider theme={supersetTheme}>
    <ThemeProvider theme={supersetTheme}>
      <ThemeProvider theme={supersetTheme}>
        ...
          <PrettiestComponentEver>
```

Ok, ok, I'll clean up :D Nice catch.

Reply:
Ahh, this one doesn't use `/spec/helpers/testing-library` because it's in the `packages` part of the monorepo, so there's no access to it. 

We should probably eventually move the helpers to `superset-ui/core` or a new testing package.